totp.go

```diff
@@ -38,13 +42,15 @@ func BarcodeImage(label string, secretkey []byte, opt *Options) ([]byte, error)
 
 	u.RawQuery = params.Encode()
 
-	c, err := qr.Encode(u.String(), qr.M)
+	c, err := qr.Encode(u.String(), qr.ECLevelM)
 
 	if err != nil {
 		return nil, err
 	}
 
-	return c.PNG(), nil
+	buf := bytes.NewBuffer([]byte{})
```

Comment:
The zero-value of bytes.Buffer is usable. i.e. this can simply be "var buf bytes.Buffer" (and add an ampersand to the next line).